lib/index.d.ts

```diff
@@ -1,4 +1,4 @@
 import type { ShowdownExtension } from "showdown";
 
-declare function showdownHighlight({ pre: Boolean }): ShowdownExtension[];
+declare function showdownHighlight({ pre: Boolean, auto_detection: Boolean }): ShowdownExtension[];
```

Comment:
Here, I want to know why `Boolean` is used. `boolean` is sufficient and desirable with modern TypeScript. Since it doesn't change any behavior for right now, I just followed previous convention. Feedback wanted.

https://stackoverflow.com/questions/64443288/what-is-the-difference-between-boolean-and-boolean-in-typescript